lib/Dialect/FIRRTL/Transforms/GrandCentral.cpp

```diff
@@ -1883,20 +1883,18 @@ void GrandCentralPass::runOnOperation() {
                              op, cast<hw::HWModuleLike>(*dut)))
                 return true;
 
+              // Lower this instance to a bind later.
               instance.getValue()->setAttr("lowerToBind", trueAttr);
-              instance.getValue()->setAttr(
-                  "output_file",
-                  hw::OutputFileAttr::getFromFilename(
-                      &getContext(),
-                      maybeExtractInfo.getValue().bindFilename.getValue(),
-                      /*excludeFromFileList=*/true));
```

Comment:
There are some knock-on effects that come from changing this that should be cleaned up:
1. Everything related to the `bindFileName` is now dead code.
2. The `ExtractGrandCentralAnnotation` should be changed here and in upstream SiFive code to no longer encode a bindfile.
3. Documentation should be updated to enshrine this change to `ExtractGrandCentralAnnotation`.

Reply:
@seldridge I believe the above bullets were addressed for CIRCT (and tests updated) in [534a3b7](https://github.com/llvm/circt/pull/3435/commits/534a3b78a779581d1fc4862c9c7cfe882ef25173).

Regarding changing upstream code to no longer encode a bindfile: I don't know exactly where to make this change, and will follow up with you, since there may be other changes upstream related to this anyway. Does the removal of `filename` from all the things producing this annotation need to be done before this patch can go in? If I understand CIRCT correctly, we had some validation on the `filename` being present on that annotation, which I removed, but will anything complain if an unexpected field is present?